crates/primitives-traits/Cargo.toml

```diff
@@ -136,17 +141,23 @@ serde = [
 	"op-alloy-consensus?/serde",
 	"k256/serde",
 	"secp256k1?/serde",
-	"alloy-trie/serde"
+	"alloy-trie/serde",
+	"scroll-alloy-consensus?/serde"
 ]
 reth-codec = [
 	"dep:reth-codecs",
 	"dep:modular-bitfield",
 	"dep:byteorder",
+	"scroll-alloy-consensus?/reth-codec"
 ]
 op = [
 	"dep:op-alloy-consensus",
 ]
-scroll = ["revm-primitives/scroll"]
+scr = ["scroll-alloy-consensus"]
```

Comment:
This is a trick used by optimism as well in order to ONLY import the `InMemorySize` implementation for `scroll-alloy-consensus::ScrollTypedTransaction`. Using the `scroll` flag to gate it causes the flag to propagate across all the workspace during linting, failing on various missing trait implementations.

Reply:
Should we name this feature something more expressive like `scroll-tx-mem`.